### ISSUE TITLE
Tailscale reporting no internet access when using exit node on Windows 11

### ISSUE THREAD

**Issue by chrisray1997** [tailscale/tailscale#8009]
### What is the issue?

I recently updated Tailscale to version 1.40.0 on my Windows 11 machine as well as upgraded to version 1.40.0 on my Linux Rasberry pi. When utilizing Tailscale on my Windows 11 machine, I am seeing an error that there is no internet connection when I am connected to Tailscale. I am using the Rasberry pi as my exit node and confirmed that the exit node works with my Mac using version 1.38.4. I have also confirmed that the internet connection on my Windows 11 machine works without issue when I am not connected to Tailscale. 

### Steps to reproduce

Upgrade Windows 11 to Tailscale version 1.40.0

### Are there any recent changes that introduced the issue?

Upgrade to version 1.40.0 on Windows 11

### OS version

Windows 11, Macos 13.3.1, Raspbian GNU/Linux 11 (bullseye)

### Tailscale version

1.40.0

### Bug report

BUG-f4d63569f4b9a3bb72ca2b001180955ee32e3d702329856460ce3e5b764890e1-20230501005450Z-5d9ffa19e911b64a

**Comment by DentonGentry** [comment]
Around the time of the bugreport:
```
2023-05-01 01:10:51.108423936 +0000 UTC: wg: [/u08b] - Handshake did not complete after 5 seconds, retrying (try 5)
2023-05-01 01:10:51.108836905 +0000 UTC: wg: [/u08b] - Sending handshake initiation
2023-05-01 01:10:56.296489924 +0000 UTC: wg: [/u08b] - Handshake did not complete after 5 seconds, retrying (try 2)
2023-05-01 01:10:56.296767164 +0000 UTC: wg: [/u08b] - Sending handshake initiation
2023-05-01 01:11:01.405138895 +0000 UTC: wg: [/u08b] - Handshake did not complete after 5 seconds, retrying (try 3)
2023-05-01 01:11:01.405577281 +0000 UTC: wg: [/u08b] - Sending handshake initiation
2023-05-01 01:11:06.491587881 +0000 UTC: wg: [/u08b] - Handshake did not complete after 5 seconds, retrying (try 4)
2023-05-01 01:11:06.491941527 +0000 UTC: wg: [/u08b] - Sending handshake initiation
2023-05-01 01:11:11.604183122 +0000 UTC: wg: [/u08b] - Handshake did not complete after 5 seconds, retrying (try 5)
2023-05-01 01:11:11.604453122 +0000 UTC: wg: [/u08b] - Sending handshake initiation
2023-05-01 01:11:16.707571618 +0000 UTC: wg: [/u08b] - Handshake did not complete after 5 seconds, retrying (try 6)
2023-05-01 01:11:16.707842504 +0000 UTC: wg: [/u08b] - Sending handshake initiation
2023-05-01 01:11:22.054384578 +0000 UTC: wg: [/u08b] - Handshake did not complete after 5 seconds, retrying (try 7)
2023-05-01 01:11:22.054744214 +0000 UTC: wg: [/u08b] - Sending handshake initiation
2023-05-01 01:11:27.363195507 +0000 UTC: wg: [/u08b] - Handshake did not complete after 5 seconds, retrying (try 8)
2023-05-01 01:11:27.363458841 +0000 UTC: wg: [/u08b] - Sending handshake initiation
2023-05-01 01:11:32.416062606 +0000 UTC: wg: [/u08b] - Handshake did not complete after 5 seconds, retrying (try 9)
2023-05-01 01:11:32.416322241 +0000 UTC: wg: [/u08b] - Sending handshake initiation
2023-05-01 01:11:37.700430407 +0000 UTC: wg: [/u08b] - Handshake did not complete after 5 seconds, retrying (try 10)
2023-05-01 01:11:37.700853167 +0000 UTC: wg: [/u08b] - Sending handshake initiation
2023-05-01 01:11:42.961778952 +0000 UTC: wg: [/u08b] - Handshake did not complete after 5 seconds, retrying (try 11)
2023-05-01 01:11:42.962010358 +0000 UTC: wg: [/u08b] - Sending handshake initiation
2023-05-01 01:11:48.244969061 +0000 UTC: wg: [/u08b] - Handshake did not complete after 5 seconds, retrying (try 12)
2023-05-01 01:11:48.245278333 +0000 UTC: wg: [/u08b] - Sending handshake initiation
2023-05-01 01:11:53.373811524 +0000 UTC: wg: [/u08b] - Handshake did not complete after 5 seconds, retrying (try 13)
2023-05-01 01:11:53.374129858 +0000 UTC: wg: [/u08b] - Sending handshake initiation
2023-05-01 01:11:58.604193068 +0000 UTC: wg: [/u08b] - Handshake did not complete after 5 seconds, retrying (try 14)
2023-05-01 01:11:58.604428433 +0000 UTC: wg: [/u08b] - Sending handshake initiation
2023-05-01 01:12:03.764993704 +0000 UTC: wg: [/u08b] - Handshake did not complete after 5 seconds, retrying (try 15)
2023-05-01 01:12:03.765291153 +0000 UTC: wg: [/u08b] - Sending handshake initiation
2023-05-01 01:12:08.888122784 +0000 UTC: wg: [/u08b] - Handshake did not complete after 5 seconds, retrying (try 16)
2023-05-01 01:12:08.888568461 +0000 UTC: wg: [/u08b] - Sending handshake initiation
2023-05-01 01:12:14.134706796 +0000 UTC: wg: [/u08b] - Handshake did not complete after 5 seconds, retrying (try 17)
2023-05-01 01:12:14.135026536 +0000 UTC: wg: [/u08b] - Sending handshake initiation
2023-05-01 01:12:19.481763429 +0000 UTC: wg: [/u08b] - Handshake did not complete after 5 seconds, retrying (try 18)
2023-05-01 01:12:19.482091867 +0000 UTC: wg: [/u08b] - Sending handshake initiation
2023-05-01 01:12:24.502343891 +0000 UTC: wg: [/u08b] - Handshake did not complete after 5 seconds, retrying (try 19)
2023-05-01 01:12:24.502731965 +0000 UTC: wg: [/u08b] - Sending handshake initiation
2023-05-01 01:12:25.560931434 +0000 UTC: wg: [IriG3] - Sending handshake initiation
2023-05-01 01:12:29.546486861 +0000 UTC: wg: [/u08b] - Handshake did not complete after 5 seconds, retrying (try 20)
2023-05-01 01:12:29.546755559 +0000 UTC: wg: [/u08b] - Sending handshake initiation
```

It appears to be attempting to contact the exit node, and not succeeding.

**Comment by chrisray1997** [comment]
I have used the exit node with my mac and my iPhone without issue. Besides updating the tailscale version on the Windows 11 machine as well as the exit node device. Nothing else has changed.

**Comment by chrisray1997** [comment]
Here is the most recent bug report from the affected Windows 11 machine BUG-e2494f227422806dfb328af7e9c4d50f5ada79e7337bec7cd90cad23cc6ef432-20230501012632Z-5ceeb7e2ea38c064

**Comment by chrisray1997** [comment]
After playing around with this I think I figured it out, at least a workaround for now until I figure out why my subnet router isn't working.

I turned off using Tail scale subnets from the Windows machine and I can reach the internet with my exit node. For some reason, using the subnet router was breaking the path out to the internet and turning it off was the trick. I confirmed that the subnet router is working for other devices (using 1.40.0) on the LAN so I am not sure what changed on the Windows client except for the update to 1.40.0.
